src/sentry/notifications/notifications/active_release.py

```diff
@@ -0,0 +1,238 @@
+from sentry.types.integrations import EXTERNAL_PROVIDERS, ExternalProviders
+from sentry.types.releaseactivity import ReleaseActivityType
+from sentry.utils import metrics
+from sentry.utils.http import absolute_uri
+
+
+class ActiveReleaseAlertNotification(AlertRuleNotification):
+    message_builder = "ActiveReleaseIssueNotificationMessageBuilder"
+    metrics_key = "release_issue_alert"
+    notification_setting_type = NotificationSettingTypes.ACTIVE_RELEASE
+    template_path = "sentry/emails/release_alert"
+
+    def __init__(
+        self,
+        notification: Notification,
+        target_type: ActionTargetType,
+        target_identifier: int | None = None,
+        last_release: Optional[Release] = None,
+    ) -> None:
+        from sentry.rules.conditions.active_release import ActiveReleaseEventCondition
+
+        super().__init__(notification, target_type, target_identifier)
+        self.last_release = (
+            last_release
+            if last_release
+            else ActiveReleaseEventCondition.latest_release(notification.event)
+        )
+
+    def get_notification_title(self, context: Mapping[str, Any] | None = None) -> str:
+        return "Active Release alert triggered"
+
+    def send(self) -> None:
+        from sentry.notifications.notify import notify
+
+        metrics.incr("mail_adapter.notify")
+        logger.info(
+            "mail.adapter.notify",
+            extra={
+                "target_type": self.target_type.value,
+                "target_identifier": self.target_identifier,
+                "group": self.group.id,
+                "project_id": self.project.id,
+            },
+        )
+
+        participants_by_provider = self.get_participants()
+        if not participants_by_provider:
+            logger.info(
+                "notifications.notification.rules.activereleasealertrulenotification.skip.no_participants",
+                extra={
+                    "target_type": self.target_type.value,
+                    "target_identifier": self.target_identifier,
+                    "group": self.group.id,
+                    "project_id": self.project.id,
+                },
+            )
+            return
+
+        # Only calculate shared context once.
+        shared_context = self.get_context()
+
+        for provider, participants in participants_by_provider.items():
+            if self.target_type == ActionTargetType.RELEASE_MEMBERS:
+                last_release = shared_context.get("last_release", None)
+                release_version = last_release.version if last_release else None
+                event = shared_context.get("event", None)
+                for participant in participants:
+                    self.record_active_release_notification_sent(
+                        participant, event, provider, release_version
+                    )
+                if (
+                    features.has("organizations:active-release-monitor-alpha", self.organization)
```

Comment:
Need to start removing these flags. Will do them in one sweep once this is in a reasonable shape. 

`projects:active-release-monitor-default-on` and `organizations:active-release-monitor-alpha` flags should, in combination, be enough to gate any users from using this.